modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
 
 . To complete changing the network plugin, reboot each node in your cluster. You can reboot the nodes in your cluster with either of the following approaches:
 
+[IMPORTANT]
+====
+The following scripts reboot all of the nodes in the cluster at the same time. This can cause your cluster to be unstable. Another option is to follow the link::https://docs.openshift.com/container-platform/4.15/nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-gracefully_nodes-nodes-rebooting[rebooting a node gracefully] procedure. The procedure reboots the nodes one-by-one so clusters with many nodes will experience considerable downtime.
```

Comment:
Draining node cannot be done when the cluster network is not ready. So a graceful node rebooting is not possible in this case. Users can reboot nodes manually, without cordoning and draining nodes.

Reply:
Would it be helpful to link to this content: https://docs.openshift.com/container-platform/4.16/nodes/nodes/nodes-nodes-rebooting.html? @pliurh

Reply:
I don't think it's helpful here. That link is about how to reboot a node gracefully. However, it does not apply in this situation.

Reply:
The link takes you to the section in the docs that are generally about "rebooting a node".